.circleci/config.yml

```diff
@@ -76,6 +76,21 @@ jobs:
             docker push janx/firefox-hg
           no_output_timeout: 30m
 
+  rust:
+    docker:
+      - image: docker:stable
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          command: |
+            cd rust
+            docker login -u $DOCKER_USER -p $DOCKER_PASS
+            docker pull janx/ubuntu-dev
+            docker build -t janx/rust -f rust.dockerfile .
+            docker push janx/rust
```

Comment:
Nit: Historically, Docker images were under my username (`janx`) but nowadays we prefer to put them under our Docker Hub org (`janitortechnology`). Please tag and push the Rust image as `janitortechnology/rust`.